src/stdlib_ascii.f90

```diff
@@ -181,31 +186,39 @@ pure logical function is_blank(c)
         character(len=1), intent(in) :: c !! The character to test.
         integer :: ic
         ic = iachar(c)             ! TAB
-        is_blank = (ic == iachar(' ')) .or. (ic == int(z'09'));
+        is_blank = ic == iachar(' ') .or. ic == int(z'09')
     end function
 
     !> Returns the corresponding lowercase letter, if `c` is an uppercase
     !  ASCII character, otherwise `c` itself.
     pure function to_lower(c) result(t)
         character(len=1), intent(in) :: c !! A character.
-        character(len=1) :: t
-        integer :: diff
-        diff = iachar('A')-iachar('a')
-        t = c
-        ! if uppercase, make lowercase
-        if (is_upper(t)) t = achar(iachar(t) - diff)
+        character(len=1)             :: t
+        integer :: k
+
+        k = index( upper_case, c )
+
+        if ( k > 0 ) then
+            t = lower_case(k:k)
+        else
+            t = c
+        endif
```

Comment:
I agree all the other changes... but I'm not quite sure about this one. The `to_lower` and `to_upper` functions got around 4x slower (0.05 s vs 0.20 s) and this change is a little bit overkill since the old one [already works with both ASCII and EBICID char tables](https://en.wikipedia.org/wiki/Letter_case#Methods_in_programming).
However a ASCII to EBICID function could be easily implemented, if desired, using a [static lookup table](https://people.cs.umu.se/isak/Snippets/a2e.c)

Reply:
I was not aware of that property of EBCDIC, but in that case, of course, the old implementation requires no replacement :).